### ISSUE TITLE
Provide an option to force the merge of plugins (even if there is just one available)

### ISSUE THREAD

**Issue by pnicolucci** [OpenLiberty/open-liberty#4260]
It should be possible to use the plugin merge utility even if there is only one plugin-cfg.xml

**Comment by pnicolucci** [comment]
Story #4261

**Comment by wtlucy** [comment]
**Summary of changes:**
The `pluginUtility merge` command no longer returns an error if just one plugin configuration file is passed to it; the command will now generate a merged plugin configuration file given just one input file.  For example, if one or more plugin configuration files exist in `/plugins`, the following command will generate a merged plugin configuration file: `pluginUtility merge --sourcePath=/plugins`.  However, if no plugin configuration files exist in `/plugins`, the command will return an error.

The error message for passing in an insufficient number of plugin configuration files (0) has been updated to `Provide at least one plug-in configuration file to do the merge.`

**Summary of testing:**
Two new functional tests have been added:

- `testPluginUtilityMergeOneFile` - invoke `pluginUtility merge --sourcePath=PLUGIN_DIR` with just one plugin configuration file in `PLUGIN_DIR` and verify that the merged configuration file is created
- `testPluginUtilityMergeAtLeastOneFile` - invoke `pluginUtility merge --sourcePath=PLUGIN_DIR` with no plugin configuration files in `PLUGIN_DIR` and verify that the correct error is returned

**Comment by sabolo** [comment]
Please can you complete the Feature Test Summary issue I have added to your Epic.  This issue is what is used to review the FAT testing and obtain the FAT focal approval.  Many thanks.

**Comment by gkwan-ibm** [comment]
***Serviceability Approval Comment*** - Please answer the following questions for serviceability approval (note that this is a new approval step -- comments welcome on how to improve it):

1.  WAD -- does the WAD identify the most likely problems customers will see and identify how the feature will enable them to diagnose and solve those problems without resorting to raising a PMR?  Have these issues been addressed in the implementation?

2.  Test and Demo -- As part of the new serviceability process we're asking feature teams to test and analyze common problem paths for serviceability and demo those problem paths to someone not involved in the development of the feature (eg. L2, test team, or another development team).  
a) What problem paths were tested and demonstrated? 
b) Who did you demo to?
c) Do the people you demo'd to agree that the serviceability of the demonstrated problem scenarios is sufficient to avoid PMRs for any problems customers are likely to encounter, or that L2 should be able to quickly address those problems without need to engage L3?

3.  SVT -- SVT team is often the first team to try new features and often encounters problems setting up and using them.  Note that we're not expecting SVT to do full serviceability testing -- just to sign-off on the serviceability of the problem paths they encountered.
a) Who conducted SVT tests for this feature?
b) Do they agree that the serviceability of the problems they encountered is sufficient to avoid PMRs, or that L2 should be able to quickly address those problems without need to engage L3?  

4.  Queues -- Which L2 / L3 queues will handle PMRs for this feature? Do the respective L2 / L3 teams know they are supporting it?

**Comment by bsbyrd1** [comment]
ID approval added, since this feature only required review of a message, which has been approved.

**Comment by pnicolucci** [comment]
All work is completed here and all approvals have been received. I'll close this epic.

**Comment by marikaj123** [comment]
Blog GA Info: https://github.ibm.com/was-liberty/WS-CD-Open/issues/13156
